lighthouse-core/test/audits/image-aspect-ratio-test.js

```diff
@@ -137,17 +137,6 @@ describe('Images: aspect-ratio audit', () => {
     },
   });
 
-  testImage('has invalid natural sizing information', {
-    rawValue: true,
-    warning: 'Invalid image sizing information https://google.com/logo.png',
-    clientSize: [100, 100],
-    naturalSize: [0, 0],
-    props: {
-      isCss: false,
-      usesObjectFit: false,
-    },
-  });
-
```

Comment:
should add a test (or modify this one) that a 1x1 pixel image passes when stretched to a different aspect ratio, to make sure we don't regress in the future on your use case :)

Reply:
@CodeDem we want to test that an image that's similar to the issue that was filed.

i.e. an image with `{clientSize: [300, 200], naturalSize: [1, 1]}` passes the audit

Reply:
@patrickhulce oh okay got it, so it will be just renaming test to` 'is placeholder image'` and updating it so that it won't expect  `warning: 'Invalid image sizing information https://google.com/logo.png'`  to pass basically removing the warning check!